### ISSUE TITLE
Somne questions when init

### ISSUE THREAD

**Issue by GLY-123** [SpatialVision/Prior-Depth-Anything#32]
class DepthAnythingV2(nn.Module):
    def __init__(
        self, 
        encoder='vitl', 
        features=256, 
        out_channels=[256, 512, 1024, 1024], 
        use_bn=False, 
        use_clstoken=False,
        encoder_cond_dim=-1
    ):
        super(DepthAnythingV2, self).__init__()
        
        self.intermediate_layer_idx = {
            'vits': [2, 5, 8, 11],
            'vitb': [2, 5, 8, 11], 
            'vitl': [4, 11, 17, 23],
            'vitg': [9, 19, 29, 39]
        }
        
        self.encoder = encoder
        self.encoder_cond_dim = encoder_cond_dim
        self.pretrained = DINOv2(model_name=encoder)
        self.out_channels = features // 2
        
        self.depth_head = DPTHead(self.pretrained.embed_dim, features, use_bn, out_channels=out_channels, 
                                  use_clstoken=use_clstoken, encoder_cond_dim=encoder_cond_dim)

    def forward(self, image, input_size=518, condition=None, device='cuda:0',return_features=False):
        x, (h, w) = self.raw2input(image, input_size, device)
        
        rh, rw = x.shape[-2:]
        patch_h, patch_w = rh // 14, rw // 14
        
        if self.encoder_cond_dim > 0: 
            condition = F.interpolate(condition, (rh, rw), mode="bilinear", align_corners=True)
        else: 
            condition = None
            
        features = self.pretrained.get_intermediate_layers(x, self.intermediate_layer_idx[self.encoder], return_class_token=True, condition=condition)
        if return_features:

            features=self.pretrained.forward_features_with_condition(x,condition=condition)["x_norm_patchtokens"]

            print(self.pretrained.embed_dim)
            print(self.pretrained.patch_embed.embed_dim)
            print(features.shape)
            return features
as the code above,using ViTL to init DepthAnythingV2 but i get the out dim is 768.Is that right?
